app/src/main/java/ru/vssemikoz/newsfeed/MainActivity.java

```diff
@@ -100,6 +113,20 @@ private void changeFavoriteState(int position) {
         }
     }
 
+    private void showNewsInBrowserByUrl(int position){
+        NewsItem item = newsItemsFromDB.get(position);
+        String urlString = item.getUrl();
+        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(urlString));
+        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+        intent.setPackage("com.android.chrome");
+        try {
+            getApplicationContext().startActivity(intent);
+        } catch (ActivityNotFoundException ex) {
+            intent.setPackage(null);
+            getApplicationContext().startActivity(intent);
+        }
+    }
+
     private void initRecViewData(){
```

Comment:
What do you mean by `Rec`? It may be confusing when you'll have recording or something aside from RecyclerView in the class. Let's rename this method ;)